erpnext/crm/doctype/prospect/prospect.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe.model.document import Document
+from frappe.model.mapper import get_mapped_doc
+
+class Prospect(Document):
```

Comment:
Since our main aim is to aggregate the lead data in prospects, we also need to. update prospect whenever a linked lead is updated. Add a method for the same triggered on lead update.